app/ts/LiquidityProvidingUI/Liquidity.tsx

```diff
@@ -215,47 +215,57 @@ export const Liquidity = ({ maybeReadClient, maybeWriteClient, universe, reputat
 	const clear = () => {
 		marketData.deepValue = undefined
 	}
+	useSignalEffect(() => {
+		selectedMarket.deepValue
```

Comment:
I feel like this line should be inside `clear()` and should have a comment why it exists.